inbc-program/inbc/parser.py

```diff
@@ -61,17 +61,6 @@ def parse_fota_args(self) -> None:
                                  help='Remote URI from where to retrieve package')
         parser_fota.add_argument('--releasedate', '-r', default='2026-12-31', required=False, type=validate_date,
```

Comment:
Same question as the one in readme file for INBC.
Just to confirm. We still use the releasedate but not rest of the parameters (e.g. vendor, manufacturer etc) is not needed?
Is that what is seen from the logs in the bug opened by validation?

Applies to all references to releasedate in the file.

Reply:
yes, we retrieve those 4 from the system itself.  Those are the ones that show in the logs that she provided.

Reply:
yes, we retrieve those 4 parameters from the system itself.  That is what is shown in the logs.